api/client/volume/client.go

```diff
@@ -4,19 +4,19 @@ import (
 	"bytes"
 	"errors"
 	"fmt"
-	"io"
-	"io/ioutil"
-	"strconv"
-
 	"github.com/libopenstorage/openstorage/api"
 	"github.com/libopenstorage/openstorage/api/client"
 	"github.com/libopenstorage/openstorage/volume"
+	"io"
+	"io/ioutil"
+	"strconv"
 )
 
 const (
 	graphPath  = "/graph"
 	volumePath = "/osd-volumes"
 	snapPath   = "/osd-snapshot"
+	credsPath  = "/osd-creds"
```

Comment:
nit: credsPath = "/" + api.OsdCredsPath

Reply:
Will leave this as it is to match the rest of the file.